src/rust/Dockerfile

```diff
@@ -153,6 +153,18 @@ RUN mkdir --parents "${TEST_DIR}"
 RUN --mount=type=cache,target="${CARGO_TARGET_DIR}",sharing=locked \
     --mount=type=cache,target="${REGISTRY_CACHE_TARGET}",sharing=locked \
     --mount=type=cache,target="${RUSTUP_CACHE_TARGET}",sharing=locked \
+<<EOF
+    # First, run cargo test without --message-format=json; this will expose
+    # errors to the developer over stdout/err in human-readable format.
+    # (These two `cargo tests` could be merged, with a jq 
```

Comment:
what does this mean? Is it suggesting an alternative implementation? It might be clearer to readers of this code to just.. not mention it? Or explain why we _don't_ do that here?

Reply:
i will remove it